admin_guide/manage_rbac.adoc

```diff
@@ -762,8 +762,158 @@ To bind the new role to a user, run:
 $ oc adm policy add-role-to-user podview user2 --role-namespace=bob-project -n bob-project
 ----
 
+[[creating-cluster-role]]
+== Creating a new cluster role
+
+To create a cluster role, run the following command:
+
+----
+$ oc create clusterrole -h ...
+----
+
+The following excerpt from the help of this command describes its usage:
```

Comment:
@kalexand-rh I'd not put help output into the docs. That's a slippery slope (why not just do it for everything, y'now?). I'd suggest summarising what it can do and maybe use one of the examples.

It might make sense to put somewhere in the docs "You can always see <command> --help for any command for more information", but that might be out of scope for this BZ.